ui/component/or-dashboard-builder/src/or-dashboard-boardsettings.ts

```diff
@@ -0,0 +1,275 @@
+                                              @or-mwc-input-changed="${() => { this.setToMobilePreset(); }}">
+                                </or-mwc-input>
+                            </div>
+                        </div>
+                    ` : null}
+                </div>
+                <!-------------------->
+                ${screenPresets.length > 1 ? html`
+                    <div>${this.generateExpandableHeader(i18next.t('display'))}</div>
+                    <div>
+                        ${this.expandedPanels.includes(i18next.t('display')) ? html`
+                            <div style="padding: 24px 24px 24px 24px;">
+                                ${this.scalingPresetTemplate(screenPresets, scalingPresets)}
+                            </div>
+                        ` : null}
+                    </div>
+                ` : undefined}
+                <!-------------------->
+                ${screenPresets.length > 1 ? html`
+                    <div>${this.generateExpandableHeader('Breakpoints')}</div>
+                    <div>
+                        ${this.expandedPanels.includes('Breakpoints') ? html`
+                            <div style="padding: 24px 24px 24px 24px;">
+                                ${this.screenPresetTemplate(screenPresets)}
+                            </div>
+                        ` : null}
+                    </div>
+                ` : undefined}
+                <!-------------------->
+            `
+        } else {
+            return html`
+                <div style="padding: 24px;">
+                    <span>${i18next.t('errorOccured')}</span><br/>
+                    <span>${i18next.t('noDashboardFound')}</span>
+                </div>
+            `
+        }
+    }
+
+    // Button to switch to a mobile preset of layout settings.
+    // TODO: Make these values not hardcoded anymore, and depend on the getDefault functions in or-dashboard-tree (that probably should move to somewhere else)
+    setToMobilePreset() {
+        if(this.dashboard?.template) {
+            this.dashboard.template.columns = 4;
+            this.dashboard.template.screenPresets![0].scalingPreset = DashboardScalingPreset.KEEP_LAYOUT;
+            this.dashboard.template.maxScreenWidth = 700;
+            this.requestUpdate();
+            this.forceParentUpdate(true);
+        }
+    }
+
+
+    /* ======================== */
+
+    // UI generation of an expandable panel header.
+    generateExpandableHeader(name: string): TemplateResult {
+        return html`
+            <span class="expandableHeader panel-title" @click="${() => { this.expandPanel(name); }}">
+                <or-icon icon="${this.expandedPanels.includes(name) ? 'chevron-down' : 'chevron-right'}"></or-icon>
+                <span style="margin-left: 6px; height: 25px; line-height: 25px;">${name}</span>
+            </span>
+        `
+    }
```

Comment:
Can `or-collapsible-panel` be used?

Reply:
That is correct! This is on the todo list. Neal wants to work with collapsible panels in his project as well, so we have plans to tweak the component a little to apply it platform-wide. (since it is not used everywhere yet)